_foundation/m365/graph-proxy.md

```
@@ -119,11 +120,11 @@ Two services are then provided:
 ![management-ui-graph-test.png]({{ site.baseurl }}/assets/images/management-ui-graph-test.png)
 
 {: .note }
->  Be sure to follow the [Installation Marketplace App Offer]({{ site.baseurl }}/bookme/marketplace-installation/) guide carefully to ensure all required resources are correctly configured, especially in multi-tenant deployments where Entra ID resources must be set up manually.
+>  Be sure to follow the [Installation Marketplace App Offer]({{ site.baseurl }}/foundation/m365/marketplace-installation/) guide carefully to ensure all required resources are correctly configured, especially in multi-tenant deployments where Entra ID resources must be set up manually.
```

markdownlint-cli2 (0.22.1)

[warning] Multiple spaces after blockquote symbol

(MD027, no-multiple-space-blockquote)